src/reducers/versions.tsx

```diff
@@ -699,6 +699,27 @@ export const getCompareInfoKey = ({
   return [addonId, baseVersionId, headVersionId, path].join('/');
 };
 
+export const createInternalCompareInfo = ({
```

Comment:
I needed this for a previous version of the patch but it's no longer needed. We might as well keep it, though. The common pattern is to always use `createInternal*` functions.